Realm/RLMSyncSession.h

```diff
@@ -56,6 +106,25 @@ NS_ASSUME_NONNULL_BEGIN
 /// session.
 - (nullable RLMSyncConfiguration *)configuration;
 
-NS_ASSUME_NONNULL_END
+/**
+ Register a progress notification block. Multiple blocks can be registered on the same session at once. All blocks
+ will be run on the same thread that they were registered on. The progress notification block will always be called
+ once immediately upon registration, in order to provide up-to-date information.
+
+ The token returned by this method must be retained as long as progress notifications are desired, and the `-stop`
+ method should be called on it when notifications are no longer needed.
+
+ If no token is returned, the notifier will not be ever be called again. There are a number of reasons this might
```

Comment:
Should "notifier" be "notification block" in this paragraph? "Notifier" would seem like the thing doing the notifying, not the thing asking to be notified.